webapp/channels/src/tests/helpers/localstorage.tsx

```diff
@@ -11,17 +13,19 @@ class LocalStorageMock {
         this.store = {};
     }
 
-    getItem(key) {
+    getItem(key: string): string | null {
         return this.store[key] || null;
     }
 
-    setItem(key, value) {
+    setItem(key: string, value: any) {
```

Comment:
would `unkown` be suitable here?

Reply:
Actually, the right type there is something like `Stringable` that doesn't exists, so I can change it to something like `{toString: () => string}`